include/SFML/Config.hpp.in

```diff
@@ -35,10 +35,25 @@
 ////////////////////////////////////////////////////////////
 // Define the SFML version
 ////////////////////////////////////////////////////////////
-#define SFML_VERSION_MAJOR      3
-#define SFML_VERSION_MINOR      0
-#define SFML_VERSION_PATCH      0
-#define SFML_VERSION_IS_RELEASE false
+// clang-format off
+#define SFML_VERSION_MAJOR @SFML_VERSION_MAJOR@
+#define SFML_VERSION_MINOR @SFML_VERSION_MINOR@
+#define SFML_VERSION_PATCH @SFML_VERSION_PATCH@
+// clang-format on
+#cmakedefine01 SFML_VERSION_IS_RELEASE
+
+
+////////////////////////////////////////////////////////////
+// Define configuration constants
+////////////////////////////////////////////////////////////
+#cmakedefine SFML_OPENGL_ES
+#cmakedefine SFML_ANDROID_USE_SUSPEND_AWARE_CLOCK
```

Comment:
Would these be better with `cmakedefine01` so they are always defined with a value?